tornado/simple_httpclient.py

```diff
@@ -517,8 +519,9 @@ def finish(self):
             # treat 302 the same as 303, and many servers use 302 for
             # compatibility with pre-HTTP/1.1 user agents which don't
             # understand the 303 status.
-            if self.code in (302, 303):
-                new_request.method = "GET"
+            if self.code in (301, 302, 303):
```

Comment:
You mean why is this tuple not pulled out as a named constant somewhere? It's only used in this one place and so I think it's clearer to just have the value here instead of having to look away somewhere else.